CONTRIBUTING.md

```diff
@@ -23,29 +26,22 @@ with it, including a sign-off) is maintained indefinitely and may be
 redistributed consistent with this project or the open source license(s)
 involved.
 
-## Quick guidelines
-
-* Keep a clean commit history. This means no merge commits, and no long series
-  of "fixup" patches (rebase or squash as appropriate). Structure work as a
-  series of logically ordered, atomic patches. `git rebase -i` is your friend.
-* Changes should be made via pull request, with review. A pull request will be
-  committed by a "committer" (an account listed in `COMMITTERS`) once it has
-  had an explicit positive review.
-* When changes are restricted to a specific area, you are recommended to add a
-  tag to the beginning of the first line of the commit message in square
-  brackets. e.g. "[uart] Fix bug #157".
-* Code review is not design review and doesn't remove the need for discussing
-  implementation options. If you would like to make a large-scale change or
-  discuss multiple implementation options, discuss on the mailing list.
-* Create pull requests from a fork rather than making new branches in
-  `github.com/lowrisc/opentitan`.
-* Do not attempt to commit code with a non-Apache license without discussing
-  first.
-* If a relevant bug or tracking issue exists, reference it in the pull request
-  and commits.
-* Do not report security vulnerabilities through public GitHub issues or pull
-  requests. For instructions on how to report vulnerabilities, please consult
-  SECURITY.md.
+### Organisational Agreement
 
-Please see [Contributing to OpenTitan](https://opentitan.org/book/doc/contributing)
-for more general guidance.
+Under the [CLA](CLA) a Grant of Copyright License is given by the individual to the Project.
+To ensure that the individual is authorised to give this grant, individuals need to be identified as
+a [Partner Individual](./docs/project_governance/useraccounts.md)
+or an [Individual Collaborator](./docs/project_governance/useraccounts.md) with an appropriate agreement.
```

Comment:
I think these links are broken and should instead be:

```suggestion
a [Partner Individual](./doc/project_governance/useraccounts.md)
or an [Individual Collaborator](./doc/project_governance/useraccounts.md) with an appropriate agreement.
```

And I think you can append `#partner-individuals` and `#individual-collaborators`, respectively, to directly link to the corresponding section.